site/documentation/content/api/tenant/index.md

```diff
@@ -258,19 +258,10 @@ The table below contains the properties which are used to configure a tenant's c
 | :------------------------| :-------: | :------------ | :------------ | :---------- |
 | *effective-since*        | *yes*     | *string*      | `-`           | The point in time at which the current settings became effective, i.e. the start of the first accounting period based on these settings. The value MUST be an [ISO 8601 compliant *combined date and time representation in extended format*](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).|
 | *max-minutes*            | *no*      | *number*      | `-1`        | The maximum connection duration in minutes allowed for the tenant for each accounting period. MUST be an integer. Minus one indicates that no limit is set. |
-| *period*                 | *no*      | *object*      | `-`          | The mode and length of an accounting period, i.e. the resource usage is calculated based on the defined number of days or on a monthly basis. |
+| *period*                 | *yes*     | *object*      | `-`         | The mode and length of an accounting period, i.e. the resource usage is calculated based on the defined number of days or on a monthly basis.  For more information, please refer to the [resource limits period]({{< relref "#resource-limits-period-configuration-format" >}}).|
```

Comment:
Since this is a new (optional) type of limits we are introducing, it is ok to define it as mandatory from a formal point of view. However, IMHO we should keep this aligned with the Data Volume limit and thus better keep it optional, also defining a fall-back default if not set explicitly. The HTTP Management API also needs to be adapted accordingly

Reply:
It makes sense to keep it optional and in sync with that of the _DataVolume_. I would update the documentation here and also the HTTP Management API to set it as optional. I would propose a separate PR for setting the default value to _monthly_.